backend/src/models/user.js

```diff
@@ -92,12 +99,13 @@ userSchema.statics.findByCredentials = async (email, password) => {
     const isMatch = await bcrypt.compare(password, user.password)
 
     if (!isMatch) {
-        throw new Error({error: "The password hashing is fucked"})
```

Comment:
whoops, this was for my personal debugging and I guess I mistakenly committed it to main 😮 you can change the error message to `either the password or email is incorrect`

Reply:
i've changed the error msg